express/src/config/environment.ts

```diff
@@ -3,6 +3,8 @@ export const googleTagId = process.env.GOOGLE_TAG_ID;
 export const showTestBanner = process.env.SHOW_TEST_BANNER === "true";
 export const region = process.env.AWS_REGION || process.env.AWS_DEFAULT_REGION;
 
+export const sessionSecret = process.env.SESSION_SECRET || "ThisIsNotASecret!OhNoNotAtAll*NotInAnyWay";
```

Comment:
```suggestion
export const sessionSecret = process.env.SESSION_SECRET;
```
The default should be undefined. In `express/src/config/session-storage.ts`, if dynamo storage is selected but no secret is provided, that should be an error.

Reply:
Sure; I'll remove the default.